examples/cookbook/recipe_one_shrink.cpp

```diff
@@ -0,0 +1,58 @@
+//////////////////////////////////////////////////////////////////////////////
+// Copyright (c) 2018, Lawrence Livermore National Security, LLC.
+// Produced at the Lawrence Livermore National Laboratory
+//
+// Created by David Beckingsale, david@llnl.gov
+// LLNL-CODE-747640
+//
+// All rights reserved.
+//
+// This file is part of Umpire.
+//
+// For details, see https://github.com/LLNL/Umpire
+// Please also see the LICENSE file for MIT license.
+//////////////////////////////////////////////////////////////////////////////
+#include "umpire/strategy/DynamicPool.hpp"
+
+#include "umpire/Allocator.hpp"
+#include "umpire/ResourceManager.hpp"
+
+#include <iostream>
+
+int main(int, char**) {
+  auto& rm = umpire::ResourceManager::getInstance();
+
+  auto allocator = rm.getAllocator("DEVICE");
+
+  /* 
+   * Create a 4 Gb pool and reserve one byte.
```

Comment:
You probably already know this, but the actual implementation will allocate more than just one byte for alignment purposes.

Reply:
Yes, I ought to change it to be explicitly a word?